Cargo.toml

```diff
@@ -101,7 +95,3 @@ name = "shell"
 
 [[example]]
 name = "process"
```

Comment:
```suggestion
name = "process"

[workspace]
members = [
  "examples/android"
]
```

Reply:
That includes it's dependency in Cargo.lock :/

Reply:
That's fine. It won't be in importers' Cargo.lock. It being in rusty_v8's lock file is desirable.

Reply:
Alrighty, done.